### ISSUE TITLE
Incorrect ETA since Navigation SDK 20

### ISSUE THREAD

**Issue by Christophe668** [mapbox/mapbox-navigation-android#1404]
Incorrect ETA displayed until user is moving.

**Android API:28**
**Mapbox Navigation SDK version:21**

### Steps to trigger behavior
``` 
		val navigationOptions = MapboxNavigationOptions.builder()
				.snapToRoute(false)
				.isFromNavigationUi(true)

		val options = NavigationViewOptions.builder()
				.navigationListener(this)
				.directionsRoute(directionsRoute)
				.directionsProfile(DirectionsCriteria.PROFILE_CYCLING)
				.waynameChipEnabled(false)
				.routeListener(createRouteListener())
				.progressChangeListener { _, progress -> }
				.navigationOptions(navigationOptions.build())
		navigationView.startNavigation(options.build())
```

### Expected behavior
Displayed ETA 30 mins and 5km

### Actual behavior
![screenshot_20181009-095154](https://user-images.githubusercontent.com/1552554/46655261-61798480-cbab-11e8-9afd-32e87b0d9815.jpg)
ETA pannel is displaying 1 min and 50 m instead of 25mins and 5km. 
In `progressChangeListener` `durationRemaining()` and `distanceRemaining()` is returning 0

**Comment by danesfeder** [comment]
Hey @Christophe668 👋 thanks for the feedback here!  We will look into this and get back to you with an answer after some investigation.

@kevinkreiser it looks like we will get `0` values from `NavigationStatus` until we are properly tracking with sufficient confidence in `Location` updates is that correct?

**Comment by guuuss** [comment]
I am having the same issue on sdk 0.20

**Comment by danesfeder** [comment]
We have found the cause of this issue and are working on a solution.

**Comment by Christophe668** [comment]
@danesfeder great news! Do you have already an estimated release date?

**Comment by Guardiola31337** [comment]
Hey @Christophe668 👋 #1437 landed fixing OP and it'll be available in the next release `0.22.0` 🎉

In the meantime, you can test using the `SNAPSHOT`. Following you can find installation instructions for `SNAPSHOT`s:

```java
repositories {
    jcenter()
    maven { url 'https://mapbox.bintray.com/mapbox' }
    maven { url "http://oss.sonatype.org/content/repositories/snapshots/" }
    // ...
}

dependencies {
    // ...
    implementation 'com.mapbox.mapboxsdk:mapbox-android-navigation:0.22.0-SNAPSHOT'
    implementation 'com.mapbox.mapboxsdk:mapbox-android-navigation-ui:0.22.0-SNAPSHOT'
    // ...
}
```

Thanks a lot!

**Comment by Guardiola31337** [comment]
Hey @Christophe668 👋 friendly reminder that [`0.22.0`](https://github.com/mapbox/mapbox-navigation-android/releases/tag/v0.22.0) version of the SDK was released yesterday 🎉 

Thanks for bringing this to us!
